### DISCUSSION TITLE
小白求助

### DISCUSSION THREAD

**Discussion by Sodamiao** [SAGIRI-kawaii/sagiri-bot#421] [Q&A] [answered]
![QQ截图20230330154019](https://user-images.githubusercontent.com/129370437/228765748-39bc33ce-f44c-4b6f-a1be-3406a5d7f53a.png)

**Comment by Sodamiao** [comment]
![QQ截图20230330161115](https://user-images.githubusercontent.com/129370437/228773023-3d1d020d-5779-42ea-bf38-0514ee51e6d5.png)
在安装依赖的时候出现了个新问题 是不是pip的原因

**Reply by SAGIRI-kawaii** [reply]
网不好

**Comment by SAGIRI-kawaii** [comment, marked as answer]
> ![QQ截图20230330154019](https://user-images.githubusercontent.com/129370437/228765748-39bc33ce-f44c-4b6f-a1be-3406a5d7f53a.png)

请仔细查看文档 [FAQ](https://sagiri-kawaii.github.io/sagiri-bot/FAQ/#pydanticerror_wrappersvalidationerror) [配置文件](https://sagiri-kawaii.github.io/sagiri-bot/configuration/#configyaml) ，或者尝试扔进翻译软件
